go/vt/wrangler/vexec.go

```diff
@@ -403,18 +406,18 @@ type ReplicationStatus struct {
 	CopyState []copyState
 }
 
-func (wr *Wrangler) getReplicationStatusFromRow(ctx context.Context, row []sqltypes.Value, primary *topo.TabletInfo) (*ReplicationStatus, string, error) {
+func (wr *Wrangler) getReplicationStatusFromRow(ctx context.Context, row sqltypes.RowNamedValues, primary *topo.TabletInfo) (*ReplicationStatus, string, error) {
```

Comment:
The changes in this function are mainly refactoring to use name references instead of index-based (and accessing the new `time_heartbeat` column)